src/Framework/MSBuildEventSource.cs

```diff
@@ -599,6 +599,18 @@ public void OutOfProcSdkResolverServiceRequestSdkPathFromMainNodeStop(int submis
             WriteEvent(80, submissionId, sdkName, solutionPath, projectPath, success, wasResultCached);
         }
 
+        [Event(81, Keywords = Keywords.All)]
+        public void MSBuildServerBuildStart(string commandLine)
+        {
+            WriteEvent(81, commandLine);
+        }
+
+        [Event(82, Keywords = Keywords.All)]
+        public void MSBuildServerBuildStop(string commandLine)
```

Comment:
To better understand performance related to using msbuild server can we add following:
- int countOfConsoleMesages
- long sumSizeOfConsoleMesages // in chars
- string clientExitType // _exitResult.MSBuildClientExitType.ToString()
- string serverExitType // _exitResult.MSBuildAppExitTypeString